src/stdatamodels/model_base.py

```diff
@@ -156,6 +156,8 @@ def __init__(self, init=None, schema=None, memmap=False,
         # Provide the object as context to other classes and functions
         self._ctx = self
 
+        self._asdf = AsdfFile()
```

Comment:
Can you add a little note explaining why this is necessary?  I imagine we'll be refactoring this code at some point in the medium term future and it might save us a head-scratch.